docs/reference/commandline/swarm_init.md

```diff
@@ -22,6 +22,7 @@ Initialize a swarm
 
 Options:
       --advertise-addr string           Advertised address (format: <ip|interface>[:port])
+      --datapath-addr string            Address or interface to use for data path traffic in the cluster (format: <ip|interface>)
```

Comment:
The usage info here should match what's printed by `swarm init --help`. I believe the flags are sorted alphabetically in `swarm init --help`.